utils/utils.go

```diff
@@ -10,6 +10,8 @@ import (
 	sentry "github.com/getsentry/sentry-go"
 )
 
+var VersionForSentry string
```

Comment:
The naming itself is wrong, are we using this version sentry..? just keep it keployVersion.